examples/predictors/lstm/predict.py

```diff
@@ -11,7 +11,7 @@
 
 # If you'd like to use a model, copy it to "trained_model_weights.h5"
 # or change this MODEL_FILE path to point to your model.
-MODEL_WEIGHTS_FILE = os.path.join(ROOT_DIR, "trained_model_weights.h5")
+MODEL_WEIGHTS_FILE = os.path.join(ROOT_DIR, "models", "trained_model_weights.h5")
```

Comment:
The notebook saves to the /models directory. Now pointing to it by default

Reply:
The `models` directory needs to be created at some point. I propose we ship it with `sample-trained_model_weights.h5` already in there, and remove `sample-trained_model_weights.h5` from its other location.

Reply:
I can include this as part of my PR for making the prescriptors work